src/openrct2/windows/top_toolbar.c

```diff
@@ -829,7 +840,16 @@ static void window_top_toolbar_paint(rct_window *w, rct_drawpixelinfo *dpi)
 			gfx_draw_sprite(dpi, SPR_G2_HYPER_ARROW, x + 5 + i * 6, y + 15, 0);
 		}
 	}
-
+	
+	//Draw Mute Button
+	if (window_top_toolbar_widgets[WIDX_FASTFORWARD].type != WWT_EMPTY) {
```

Comment:
Seems like a copy-paste error. `WIDX_FASTFORWARD` should be replaced with `WIDX_MUTE`.